Widgets/Widgets/TopReadWidget.swift

```diff
@@ -323,8 +353,7 @@ struct TopReadOverlayView: View {
 	var content: some View {
 		VStack(alignment: .leading) {
 			if isExpandedStyle {
-				// TODO: Localize
-				Text("\(currentViewCountOrEmpty) readers")
+				Text(currentViewCountOrEmpty)
```

Comment:
As per the designs, I believe this expanded style overlay (meaning, no image in the small widget family) still needs to include the "readers" text segment: `Text("\(currentViewCountOrEmpty) readers")`

<img width="180" alt="widget_small_overlay" src="https://user-images.githubusercontent.com/5652327/92966367-3f8a7b80-f42c-11ea-967b-796482dc2b1a.png">

Reply:
@staykids This is still working for me, are you seeing issues when you run? `currentViewCountOrEmpty` included "readers" via the localized string. I made a commit updating the variable names to be more clear, hopefully that helps.

Reply:
My bad, you're right - I missed that. Thanks for updating my vague variable name.